ts/test/automation/delete_account.spec.ts

```diff
@@ -44,6 +46,8 @@ test('Delete account from swarm', async () => {
   await typeIntoInput(restoringWindow, 'display-name-input', userA.userName);
   // Click continue
   await clickOnTestIdWithText(restoringWindow, 'continue-session-button');
+  console.log('sleeping for 20000ms');
```

Comment:
Lovely 🙉

Reply:
haha, yeah I basically want the app to have the time to fetch any messages from the swarm